runtime/runtime.rs

```diff
@@ -98,6 +114,10 @@ pub async fn run_js_module(
           "js/98_global_scope.js",
           "js/99_main.js",
         ))
+        .state(move |state| {
+          state.put(ZinniaPermissions {});
```

Comment:
Before this, I got an error like this:

```
thread 'fetch_tests' panicked at 'required type ZinniaPermissions is not present in GothamState container', /Users/julian/.cargo/registry/src/github.com-1ecc6299db9ec823/deno_core-0.171.0/gotham_state.rs:82:3
```

I found this to match, so just copied the strategy: https://github.com/denoland/deno/blob/34bfa2cb2c1f0f74a94ced8fc164e81cc91cb9f4/runtime/web_worker.rs#L377

Reply:
See also https://docs.rs/gotham/latest/gotham/state/index.html

Reply:
Makes sense. Thank you for the pointers.

I was surprised because we already use `ZinniaPermissions` for `deno_web`, but we didn't need to store it in the Gotham state. 🤷🏻

It's great to see that the tests are already helpful, even in the current minimal form.